### PR TITLE
Delete Strings no longer used

### PR DESCRIPTION
**Pull request by matiasbzurovski**
Delete unused Strings after those added in https://github.com/radixdlt/apps-localization-src/pull/350

### CODE REVIEW

File: Content/DerivePublicKeys/derivePublicKeys.json [hunk 1]
```diff
@@ -1,11 +0,0 @@
-{
-	"derivePublicKeys_titleCreateAccount": "Creating Account",
-	"derivePublicKeys_titleCreatePersona": "Creating Persona",
-	"derivePublicKeys_titleImportLegacyAccount": "Deriving Accounts",
```
Comment (giannis-rdx) [inline Content/DerivePublicKeys/derivePublicKeys.json:4]: you can also remove 
`accountRecoveryScan_derivingAccounts`

File: Content/Signing/signing.json [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-	"signing_useLedgerLabel_Ledger": "Ledger",
-	"signing_useLedgerLabel_lastUsed": "Last used",
-	"signing_useLedgerLabel_addedOn": "Added on",
-
-	"signing_signatureRequest_title": "Signature Request",
-	"signing_signatureRequest_body": "Make sure the following **Ledger hardware wallet device** is connected to a computer with a linked Radix Connector browser extension, and the Radix Babylon app is launched on the device.",
-	"signing_signatureRequest_instructions": "Review and sign the transaction on the Ledger device to continue.",
-
-	"signing_signatureSuccessful_title": "Signed",
-	"signing_signatureSuccessful_body": "Signature successful"
```
Comment (giannis-rdx) [inline Content/Signing/signing.json:14]: we use these two in our codebase, but I want to ensure that we are actually showing them on the screen

File: Content/DerivePublicKeys/derivePublicKeys.json (hunk 1 above)
Comment (matiasbzurovski) [reply Content/DerivePublicKeys/derivePublicKeys.json:4]: done

File: Content/Signing/signing.json (hunk 2 above)
Comment (matiasbzurovski) [reply Content/Signing/signing.json:14]: let me know, I don't think there are any screens that have these Strings.

Comment (giannis-rdx) [reply Content/Signing/signing.json:14]: yes I can confirm wallet doesn't present these strings on the UI.

**Pull request merged**